docs/community_resources/faq.md

```diff
@@ -0,0 +1,126 @@
+# FAQ
```

Comment:
=> `Support & Maintenance FAQ` or similar here and on `mkdocs.yml` link to be inline with intro.
Or is intention to rather change intro to be more generic for a generic FAQ?